src/app/pages/plan-cuentas/gestion-vigencias/cierre-vigencia/cierre-vigencia.component.ts

```diff
@@ -45,51 +45,63 @@ export class CierreVigenciaComponent implements OnInit {
   fuenteInfo: any;
 
   constructor(
-    private translate:      TranslateService,
-    private popManager:     PopUpManager,
-    private CVHelper:       CierreVigenciaHelper,
+    private translate: TranslateService,
+    private popManager: PopUpManager,
+    private CVHelper: CierreVigenciaHelper,
     private vigenciaHelper: VigenciaHelper,
-    private router:         Router,
-    private route:          ActivatedRoute,
+    private router: Router,
+    private route: ActivatedRoute
```

Comment:
A no ser que Husky ladre o no compile, mirar si se puede dejar la coma
```suggestion
    private route: ActivatedRoute,
```

Reply:
Se realiza un prepush y todo funciona, se deja el cambio